src/api.js

```diff
@@ -0,0 +1,28 @@
+var url = 'https://en.wikipedia.org/w/api.php';
```

Comment:
Let's modernize this file by changing the `var` statements to either `const` or `let` statements, depending on whether you need to assign to them more than once. Thanks!